base_station/gui/src/components/Chlorophyll.vue

```diff
@@ -19,8 +19,18 @@
       <span v-if="UVLED == 0" class="toggle__label" >UV LEDs Off</span>
 
       <input type="checkbox" id="toggle_button">
-      <span class="toggle__switch" v-if="UVLED == 0" v-on:click="UVLED=1, setPart(mosfetIDs.UVLED, true)"></span>
-      <span class="toggle__switch" v-if="UVLED == 1" v-on:click="UVLED=0, setPart(mosfetIDs.UVLED, false)"></span>
+      <span class="toggle__switch" v-if="UVLED == 0" v-on:click="UVLED=1, UVshutdown(true)"></span>
+      <span class="toggle__switch" v-if="UVLED == 1" v-on:click="UVLED=0, UVshutdown(false)"></span>

```

Comment:
The above two lines could be condensed to one line if we use the value of UVLED in the on:click= part